packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -168,6 +168,22 @@ LinkButton = () => (
 );
 ```
 
+## Inline link as span
```

Comment:
This should be an h3. Change to `### Inline link as span` to fix the "Untitled example":
![image](https://user-images.githubusercontent.com/47335686/94021018-8c9e1400-fd81-11ea-8cf3-622bdecb125f.png)

Reply:
Updated - thanks!